endpoints/_includes/functions.php

```diff
@@ -746,6 +748,30 @@ function helplineSearch($latitude, $longitude)
     return json_decode(get($search_url));
 }
 
+function getBMLTserver($latitude, $longitude, $search_state)
+{
+	$helpline_search_radius=setting('helpline_search_radius');
+	if ($search_state=="1") {
+		if (setting('tomato_helpline_fallback')) {
+			$bmlt_search_endpoint=setting('tomato_url')."/client_interface/json/?switcher=GetSearchResults&data_field_key=longitude,latitude,root_server_uri&sort_results_by_distance=1&long_val={LONGITUDE}&lat_val={LATITUDE}&geo_width=" . $helpline_search_radius;
+			$search_url = str_replace("{LONGITUDE}", $longitude, str_replace("{LATITUDE}", $latitude, $bmlt_search_endpoint));
+			$search_results=json_decode(get($search_url));
+			$pinged_BMLT_root=$search_results[0]->root_server_uri;
+			$bmlt_root=$GLOBALS['bmlt_root_server']."/";
+			if($pinged_BMLT_root!=$bmlt_root) { // If the rootserver handling the phonecall is not the local rootserver, enable tomato routing - otherwise handle call routing locally.
+				$GLOBALS['settings_whitelist']['tomato_helpline_routing']['default']=true;
+				} 			
+		}
+	} else if ($search_state=="2") {
+			if (setting('tomato_meeting_search')) {
+				return setting('tomato_url');
+			} else {
+				return $GLOBALS['bmlt_root_server'];
+			}
+		}
+	return getHelplineBMLTRootServer();
```

Comment:
wrap in an `else` clause to make it more readable.

Reply:
Will move it up into the volunteer search section and add additional else. The reason for putting it outside of the if structure was because in the case of helpline volunteer, regardless of tomato on or off, the return was the same and the only difference was changing the tomato_helpline_enabled setting. Original thought was simpler code. Changed as requested.